ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -80,26 +80,6 @@ public enum TableViewCellBackgroundStyleType: Int {
 // MARK: - Table Colors
 
 public extension Colors {
-    internal struct Table {
-        struct Cell {
-            static var image: UIColor = iconSecondary
-            static var title: UIColor = textPrimary
-            static var subtitle: UIColor = textSecondary
-        }
-
-        struct HeaderFooter {
-            static var accessoryButtonText: UIColor = textSecondary
-            static var background: UIColor = .clear
-            static var backgroundDivider: UIColor = surfaceSecondary
-            static var text: UIColor = textSecondary
-            static var textDivider: UIColor = textSecondary
-            static var textLink = UIColor(light: Palette.communicationBlueShade10.color, dark: communicationBlue)
-        }
-
-        static var background: UIColor = surfacePrimary
-        static var backgroundGrouped = UIColor(light: surfaceSecondary, dark: surfacePrimary)
-    }
-
     // Objective-C support
     @objc static var tableBackground: UIColor { return surfacePrimary }
```

Comment:
all the objc colors need to be updated then?

Reply:
Yes they'd need to. I just spoke with @huwilkes regarding this and since we'd need to remove Colors at some point maybe I should go ahead and remove the objc colors here as well. Then we can update the controls to pull in the right tokens in devmain. Would that be a good idea?

Reply:
i think we still need to have some sort of objc support for few colors. but we will need to look at client codes before making that decision.

Reply:
Sounds good. I'll work on finding a way to pull them out of `Colors`.